src/test/java/org/primeframework/mvc/GlobalTest.java

```diff
@@ -1219,21 +1219,6 @@ public void get_unknownParameters() throws Exception {
 
   @Test
   public void get_url_rewrite() {
```

Comment:
You probably should document why this method exists and how we removed the handling for `__a_` parameters since it was a large security risk. That way the next person in the code knows the history and doesn't delete this test.